rpcs3/Emu/RSX/Overlays/overlay_controls.h

```diff
@@ -185,6 +190,7 @@ namespace rsx
 			virtual ~overlay_element() = default;
 
 			virtual void refresh();
+			virtual bool is_compiled() { return m_is_compiled; }
```

Comment:
const

Reply:
It can't be const because the override will set it to false if necessary